client/src/client_sync/v28.rs

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! A JSON-RPC client for testing against Bitcoin Core `v27`.
+//!
+//! We ignore option arguments unless they effect the shape of the returned JSON data.
+
+use bitcoin::address::{Address, NetworkChecked};
+use bitcoin::{Amount, Block, BlockHash, Txid};
+
+use crate::client_sync::{handle_defaults, into_json};
+use crate::json::v28::*;
+
+crate::define_jsonrpc_minreq_client!("v27");
+
+// == Blockchain ==
+crate::impl_client_v17__getblockchaininfo!();
+crate::impl_client_v17__getbestblockhash!();
+crate::impl_client_v17__getblock!();
+crate::impl_client_v17__gettxout!();
+
+// == Control ==
+crate::impl_client_v17__stop!();
+
+// == Generating ==
+crate::impl_client_v17__generatetoaddress!();
+
+// == Network ==
+crate::impl_client_v17__getnetworkinfo!();
+crate::impl_client_check_expected_server_version!({ [270000, 270100] });
```

Comment:
```suggestion
crate::impl_client_check_expected_server_version!({ [280000] });
```
Did you run the tests against a local build of Bitcoin Core v28? This test should have failed if so.